ui/pages/bridge/prepare/prepare-bridge-page.tsx

```diff
@@ -148,16 +148,9 @@ export const useEnableMissingNetwork = () => {
         if (isPopularNetwork) {
           const isNetworkEnabled = enabledNetworkKeys.includes(chainId);
           if (!isNetworkEnabled) {
-            const enabledEvmNetworks = enabledNetworkKeys.filter((key) =>
-              FEATURED_NETWORK_CHAIN_IDS.includes(key as Hex),
-            );
-            const newNetworkEnabledEvmNetworks = [
-              chainId,
-              ...enabledEvmNetworks,
-            ];
-            dispatch(
-              setEnabledNetworks(newNetworkEnabledEvmNetworks, namespace),
-            );
+            // Bridging between popular networks indicates we want the 'select all' enabled
+            // This way users can see their full briding tx activity
+            dispatch(enableAllPopularNetworks());
```

Comment:
Bridging to tokens that you have not selected in the network manager won't show up in token list/TX Activity.

So we ensure that we change the token filters so that you can see the new tokens you've swapped to inside the token list and activity tabs.